presentation/src/main/java/me/androidbox/presentation/navigation/NavigationGraph.kt

```diff
@@ -137,7 +135,7 @@ fun NavigationGraph(
                         Screen.EventScreen.EVENT_SCREEN
                     }
                     is Task -> {
-                        Screen.TaskDetailScreen.TASK_DETAIL_SCREEN
+                        TASK_DETAIL_SCREEN
```

Comment:
In one you are using
 Screen.EventScreen.EVENT_SCREEN
and in the other just  TASK_DETAIL_SCREEN

It would make more sense if you unify it

Either
EVENT_SCREEN and TASK_DETAIL_SCREEN

Or
Screen.EventScreen.EVENT_SCREEN and Screen.TaskDetailScreen.TASK_DETAIL_SCREEN


Also
Shouldn't the Event one be "EVENT_DETAIL_SCREEN" ?

Reply:
@MKiperszmid updated